src/main.rs

```diff
@@ -240,16 +361,27 @@ async fn run(opts: Command) -> anyhow::Result<()> {
         qps_sum / num_random_read_workers as f64,
     );
 
+    if let SubCommand::Verify { kill_opts: Some(KillOpts { kill_script, .. }), .. } = &opts.sub {
+        let _ = tokio::process::Command::new("sh").arg("-c").arg(kill_script).spawn();
+    }
+
```

Comment:
is this the last execution of the kill script? if that's the case, would it be better to gracefully terminate the server?

Reply:
I guess techincally yes, but this is just a test anyway so I would say we don't really care what happens to the server.